recipes/scikit-surprise/LICENSE.md

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2016, Nicolas Hug
```

Comment:
Is the library not included in the tarball?

Reply:
You mean the license file? Yes, it is. Should I remove it from the recipe?

Reply:
Yes, sorry the license. If it's in the tarball then you don't need it in the recipe